vt_scan.py

```diff
@@ -557,6 +557,9 @@ def main():
     config = {}
     results = {}
 
+    if not options.path_to_file:
```

Comment:
I think rather than doing this we should go for argparse module as explained in [PEP389](https://www.python.org/dev/peps/pep-0389/), optparse offer less possibilities (including making an option mandatory) and is now deprecated.